posthog/tasks/usage_report.py

```diff
@@ -591,6 +593,34 @@ def get_teams_with_survey_responses_count_in_period(
     return results
 
 
+@timed_log()
+@retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
+def get_teams_with_data_warehouse_rows_synced_in_period(begin: datetime, end: datetime) -> List[Tuple[int, int]]:
+    team_to_query = 1 if get_instance_region() == "EU" else 2
+
+    # dedup by job id incase there were duplicates sent
+    results = sync_execute(
+        """
+        SELECT team, sum(rows_synced) FROM (
+            SELECT JSONExtractString(properties, 'job_id') AS job_id, distinct_id AS team, any(JSONExtractInt(properties, 'count')) AS rows_synced, any(JSONExtractString(properties, 'start_time')) AS start_time
+            FROM events
+            WHERE team_id = %(team_to_query)s AND event = 'external data sync job' AND parseDateTimeBestEffort(JSONExtractString(properties, 'start_time')) BETWEEN %(begin)s AND %(end)s
```

Comment:
curious, any time zone shenanigans to worry about here?

Reply:
everything should be UTC